db/migrate/20210112221033_add_foreign_key_constraint_to_active_storage_attachments_for_blob_id.active_storage.rb

```diff
@@ -0,0 +1,10 @@
+# This migration comes from active_storage (originally 20180723000244)
+class AddForeignKeyConstraintToActiveStorageAttachmentsForBlobId < ActiveRecord::Migration[6.0]
+  def up
+    return if foreign_key_exists?(:active_storage_attachments, column: :blob_id)
+
+    if table_exists?(:active_storage_blobs)
```

Comment:
I'm not aware of this table existing, so probably the migration isn't needed (?).